lib/objectid.js

```diff
@@ -0,0 +1,12 @@
+var ObjectId;
+
+try {
+  ObjectId = require('mongodb').ObjectId;
+} catch(e) {
+  if (e.code === 'MODULE_NOT_FOUND')
+    throw e;
```

Comment:
This was suggested by @ritch in a [previous comment in this issues history](https://github.com/strongloop/loopback-datasource-juggler/pull/778#discussion_r46341514). I think it was for uses cases where mongodb is not available (ie. the browser).